_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Our First Game Jam!"
+date:   2019-04-14 14:55:00 -0500
+categories: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/ferryman-glass.png
+---
+_48 hour sprint to Unity greatness_
+[![Ferryman image]({{ "/assets/ferryman-glass.png" | relative_url }}){:class="excerpt-image"}]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %})
+<!-- Ends the excerpt text, it includes the image -->
+
+* Start Friday.
+* Build a game that is loosely based on the theme of "Spirits".
+* Submit on Sunday.
+
+Let RJ set the scene
+> You're sitting at long IKEA table in a studio apartment in Somerville, MA with your two bros, taking turns dipping a baguette into a container of chive cream cheese. Because that's just how it works."
+
+
+This was the first time we've joined a challenge like this, so obviously we had to further increase the difficulty by forcing ourselves to build and publish it in Unity, which we only have a few tutorials' worth of experience with.
+We broke up into 3 roles for the most part: Aaron on art, Sam building the C# game logic, and RJ tying everything together in Unity. We each learned a lot about our roles, and how different specializing like that was from our experiences building our other games.
+
+
+## Unity
+
+We started out worried about how we would collaborate on the game, and in such a rapid-fire way that the time constraints demanded.
```

Comment:
maybe a small subtitle "Collaboration" here, and "Layout" for the next paragraph